Tests/SonarQube.Bootstrapper.Tests/ArgumentProcessorTests.cs

```diff
@@ -37,7 +37,49 @@ public void ArgProc_UnrecognizedArgumentsAreIgnored()
 
             // 1. Minimal command line settings with extra values
             IBootstrapperSettings settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo", "foo", "blah", "/xxxx");
-            AssertExpectedUrl("foo", settings);
+            AssertCommonSettings(settings, "foo", true, "\"/d:sonar.host.url=foo\" \"foo\" \"blah\" \"/xxxx\"");
+        }
+
+        [TestMethod]
+        public void ArgProc_InstallTargets()
+        {
+            TestLogger logger = new TestLogger();
+
+            //  No argument passed -> install targets
+            IBootstrapperSettings settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo");
+            AssertCommonSettings(settings, "foo", true, "\"/d:sonar.host.url=foo\"");
```

Comment:
The 3rd param is to check the ChildCmdLineArgs to add coverage over RemoveBootstrapperArgs